pil-analyzer/tests/condenser.rs

```diff
@@ -99,7 +99,7 @@ fn create_constraints() {
     y = x_is_zero + 2;
     "#;
     let expected = r#"namespace N(16);
-    let force_bool: expr -> std::prelude::Constr = |c| c * (1 - c) = 0;
+    let force_bool: expr -> std::prelude::Constr = |c| c * (1_expr - c) = 0_expr;
```

Comment:
Oh, this is just the printed version, `1` is still allowed by the parser?

Reply:
What would it take for the compiler to remember whether the original literal had a suffix or not so it can print it the same way? Or is this a bad idea?

Reply:
We also add types for all variables and print them, and we specialize all generic function calls by adding `::<...>`. I think it's good to add the suffixes everywhere.